testing/regress/ecl/parallelchild.ecl

```diff
@@ -0,0 +1,42 @@
+/*##############################################################################
+
+    HPCC SYSTEMS software Copyright (C) 2012 HPCC Systems.
```

Comment:
ok.. but shouldn't all our copyright banners be updated annually?
(there are loads that are < 2015)